vuepress/docs/next/tutorials/samples/install-standard-demo.md

```diff
@@ -40,40 +51,42 @@ ent bundler from-git -d -r https://github.com/entando-samples/standard-demo-cont
    ![InstallPlan.png](./images/InstallPlan.png)
 
-Select `Update All` in the upper right after making your selections.
 
-6. Each installation can take several minutes, while the application downloads the Linux images for the microservices and installs the related assets.
-   ![Installed.png](./images/Installed.png)
 
-:::warning 
-(Entando 6.3.2) There is a cache issue when initially deploying the `sd-content` bundle, where not all widgets or MFEs appear on some pages, particularly the Dashboard page. 
 
-Restarting the quickstart-server pod (which contains the Entando App Engine) will clear the cache, and can be achieved with `ent k delete pod/<YOUR QUICKSTART-SERVER POD>`, e.g. `ent k delete pod/quickstart-server-deployment-5d785b997c-r4sc8`. It will take several minutes for the pod to redeploy after deletion. 
 
-Note that a restart is only applicable to the first install. 
-:::
+5. Access the Standard Banking Demo via one of the following options:
 
-7. (Option 1) If you'd like to make the Standard Banking Demo your default home page, go to `App Builder → Pages → Settings`. In
+**Option 1** If you'd like to make the Standard Banking Demo your default home page, go to `App Builder → Pages → Settings`. In
    the dropdown for Home Page, select `Home / Home SD` and click `Save`.
    ![HomepageSelect.png](./images/HomepageSelect.png)
 
 You can now navigate to your application's home page using the home icon in the upper right of the App Builder.
+
+
+**Option 2** Alternatively, you can view the Standard Banking Demo home page by going to `Pages → Management`, finding `Home SD` in the page tree, and clicking `View Published Page` from its actions.
+
 ![Homepage.png](./images/Homepage.png)
 
-8. (Option 2) Alternatively, you can view the Standard Banking Demo home page by going to `Pages → Management`,
-   finding `Home SD` in the page tree, and clicking `View Published Page` from its actions.
+:::warning 
+(Entando 6.3.2) There can be a cache issue when initially deploying the `sd-content` bundle, where not all widgets or MFEs appear on some pages, particularly the Dashboard page. 
 
-:::tip 
-There are numerous assets installed as part of the Standard Banking Demo. Entando Bundles can include more or less components, depending on objectives. It is recommended that organizations develop guidelines for bundle sizing that fit the goals of their applications and teams.
+To clear the cache, select `Adniminstration` from the bottom of the left menu, then `Reload configuration`.
+
+Alternatively, restarting the quickstart-server pod (which contains the Entando App Engine) will also clear the cache, and can be achieved with `ent k delete pod/<YOUR QUICKSTART-SERVER POD>`, e.g. `ent k delete pod/quickstart-server-deployment-5d785b997c-r4sc8`. It will take several minutes for the pod to redeploy after deletion. 
+
+Note that needing to clear the cache is only applicable to the first install. 
```

Comment:
Can this note be included in the warning above to streamline the information?

Reply:
i can do that :)